core/summary_api.php

```diff
@@ -92,6 +96,85 @@ function summary_helper_get_developer_label ( $p_user_id ) {
 }
 
 /**
+ * Calculate bug status count according to  'open', 'resolved' and 'closed', 
+ * then put the numbers into $p_cache array
+ *
+ * @param array &$p_cache    The cache array.
+ * @param string $p_key      The key of the array.
+ * @param string $p_status   The status of issues.
+ * @param string $p_bugcount The bug count of $p_status issues.
+ * @return void
+ */
+function summary_helper_build_bugcount( &$p_cache, $p_key, $p_status, $p_bugcount ) {
+	$t_resolved_val = config_get( 'bug_resolved_status_threshold' );
+	$t_closed_val = config_get( 'bug_closed_status_threshold' );
+
+	if( $t_closed_val <= $p_status ) {
+		if( isset( $p_cache[$p_key]['closed'] ) ) {
+			$p_cache[$p_key]['closed'] += $p_bugcount;
+		} else {
+			$p_cache[$p_key]['closed'] = $p_bugcount;
+		}
+	} else if( $t_resolved_val <= $p_status ) {
+		if( isset( $p_cache[$p_key]['resolved'] ) ) {
+			$p_cache[$p_key]['resolved'] += $p_bugcount;
+		} else {
+			$p_cache[$p_key]['resolved'] = $p_bugcount;
+		}
+	} else {
+		if( isset( $p_cache[$p_key]['open'] ) ) {
+			$p_cache[$p_key]['open'] += $p_bugcount;
+		} else {
+			$p_cache[$p_key]['open'] = $p_bugcount;
+		}
+	}
+}
+/** 
+ * Build bug links for 'open', 'resolved' and 'closed' issue counts
+ * 
+ * @param string $p_bug_link 		The base bug link.
+ * @param string &$p_bugs_open 		The open bugs count, return open bugs link.
+ * @param string &$p_bugs_resolved 	The resovled bugs count, return resolved bugs link.
+ * @param string &$p_bugs_closed   	The closed bugs count, return closed bugs link.
+ * @param string &$p_bugs_total   	The total bugs count, return total bugs link.
+ * @return void 
+ */
+function summary_helper_build_buglinks( $p_bug_link, &$p_bugs_open, &$p_bugs_resolved, &$p_bugs_closed, &$p_bugs_total) {
+	$t_resolved_val = config_get( 'bug_resolved_status_threshold' );
+	$t_closed_val = config_get( 'bug_closed_status_threshold' );
+
+	if( 0 < $p_bugs_open ) {
+		$p_bugs_open = $p_bug_link . '&amp;' . FILTER_PROPERTY_HIDE_STATUS . '=' . $t_resolved_val . '">' . $p_bugs_open . '</a>';
+	}
+	if( 0 < $p_bugs_resolved ) {
+		$p_bugs_resolved = $p_bug_link . '&amp;' . FILTER_PROPERTY_STATUS . '=' . $t_resolved_val . '&amp;' . FILTER_PROPERTY_HIDE_STATUS . '=' . $t_closed_val . '">' . $p_bugs_resolved . '</a>';
+	}
+	if( 0 < $p_bugs_closed ) {
+		$p_bugs_closed = $p_bug_link . '&amp;' . FILTER_PROPERTY_STATUS . '=' . $t_closed_val . '&amp;' . FILTER_PROPERTY_HIDE_STATUS . '=' . META_FILTER_NONE . '">' . $p_bugs_closed . '</a>';
+	}
+	if( 0 < $p_bugs_total ) {
+		$p_bugs_total = $p_bug_link . '&amp;' . FILTER_PROPERTY_HIDE_STATUS . '=' . META_FILTER_NONE . '">' . $p_bugs_total . '</a>';
+	}	
+}
+
+/**
+ * Calculate bug ratio 
+ * @param integer $p_bugs_open 		The open bugs count.
```

Comment:
Fix indentation of parameter description.

Reply:
Indentation fixed.